doc/source/structures/vessels/vessel.rst

```diff
@@ -363,3 +365,17 @@ All vessels share a structure. To get a variable referring to any vessel you can
     :return: :struct:`List` of :struct:`Part` objects
 
     nametag on them of any sort that is nonblank. For more information, see :ref:`ship parts and modules <parts and partmodules>`.
+
+.. attribute:: Vessel:CREWCAPACITY
+
+    :type: scalar
+    :access: Get only
+
+    crew capacity of this vessel
+
+.. method:: Vessel:CREW()
+
+    :return: :struct:`List` of :struct:`CrewMember` objects
+
+    list of all kerbonauts aboard this vessel
```

Comment:
I'd recommend making this a clickable link pointing to the CrewMember structure documentation page, as in:

"List of all {this is a link: kerbonauts} abord this vessel.":